README.md

```diff
@@ -128,15 +135,38 @@ For reference, here is a exp of a step-by-step evaluation:
 ## Problem Seven
 
 Write a step-by-step evaluation for the following expression (remember order of operations): `4 / 2 + 8 / 4`.
-
+```js
+4 / 2 + 8 / 4
+-> 2 + 8 / 4
+-> 2 + 2
+-> 4
+```
 ## Problem Eight
 
 Write a step-by-step evaluation for the following expression: `'ca' + 'ter' + 'pi' + 'llar'`.
-
+```js
+'ca' + 'ter' + 'pi' + 'llar'
+-> 'cater' + 'pi' + 'llar'
+-> 'caterpi' + 'llar'
+-> 'caterpillar'
+```
 ## Problem Nine
 
 Write a step-by-step evaluation for the following expression: `2 * 4 === 8 && 'car' + 'pool' === 'carpool'`.
-
+```js
+2 * 4 === 8 && 'car' + 'pool' === 'carpool'
+-> 8 === 8 && 'car' + 'pool' === 'carpool'
+-> true && 'car' + 'pool' === 'carpool'
+-> true && 'carpool' === 'carpool'
+-> true && true
+-> true
+```
 ## Problem Ten
 
 Write a step-by-step evaluation for the following expression: `'1' + '2' + '3' - '1'`.
+```js
+'1' + '2' + '3' - '1'
+-> '12' + '3' - '1'
+-> '123' - '1'
+->  '122'
```

Comment:
So close! last line here should be a number. The subtraction casts the type from string to number.